src/lio/warmer_helpers.h

```diff
@@ -0,0 +1,220 @@
+/*
+   Copyright 2016 Vanderbilt University
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+//***********************************************************************
+// Warmer helper definitions
+//***********************************************************************
+
+
+#ifndef _WARMER_H_
+#define _WARMER_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define WFE_SUCCESS   1
+#define WFE_FAIL      2
+#define WFE_WRITE_ERR 4
+
+typedef struct {
+    int  state;
+    int  nfailed;
+    char name[1];
+} __attribute__((packed)) db_inode_entry_t;
```

Comment:
I did look at it :) I didn't consider the differences between compilers and architectures to be a true issue since the in-process levelDB storage is, presumably, going to only read/written from a single host.